content/en/docs/concepts/storage/ephemeral-volumes.md

```diff
@@ -48,7 +48,7 @@ different purposes:
   kinds of Kubernetes data into a Pod
 - [CSI ephemeral volumes](#csi-ephemeral-volumes):
   similar to the previous volume kinds, but provided by special
-  [CSI drivers](https://github.com/container-storage-interface/spec/blob/master/spec.md)
+  [CSI drivers](https://kubernetes-csi.github.io/docs/drivers.html)
   which specifically [support this feature](https://kubernetes-csi.github.io/docs/drivers.html)
```

Comment:
```suggestion
  which specifically [support this feature](https://kubernetes-csi.github.io/docs/ephemeral-local-volumes.html)
```

Reply:
This change of link should be a proper link for it @akshayamadhuri WDYT?

Reply:
@AmarNathChary  i already raised another PR and its been merged i am closing this issue for now